### PR TITLE
fix: sample app

### LINKED ISSUE #2261: Update generated isar code in flutter sample
### Description

We ned to update the generated code.

Image

### PR DESCRIPTION
**Pull request by buenaflor**
- add missing network xml link for Spotlight
- report ttfd for root screen
- use isar community fork dependency instead (it's more stable since it's being actively maintained and the original isar dep doesn't work in android release for some reason with compileSdk 34)
- re-generate isar sample model
- fix android sample not working for android studio 2024.2.1

#skip-changelog

Closes #2261

### CODE REVIEW

File: flutter/example/lib/isar/user.g.dart [hunk 1]
```diff
@@ -13,17 +13,17 @@ extension GetUserCollection on Isar {
   IsarCollection<User> get users => this.collection();
 }
 
-final UserSchema = CollectionSchema(
+const UserSchema = CollectionSchema(
   name: r'User',
-  id: BigInt.parse("-7838171048429979076").toInt(),
+  id: -7838171048429979076,
```
Comment (buenaflor) [inline flutter/example/lib/isar/user.g.dart]: @denrase the new isar generation doesn't work on web, see CI failure

this LoC seems to be problematic

Comment (buenaflor) [reply flutter/example/lib/isar/user.g.dart]: It looks like web should work with isar community 4.x

Comment (denrase) [reply flutter/example/lib/isar/user.g.dart]: think we had a some issue before, let me check

Comment (denrase) [reply flutter/example/lib/isar/user.g.dart]: should be fixed, same approach as before

**Comment by buenaflor** [comment]
@denrase can you approve pls

**Review by stefanosiano — APPROVED** [review]

**Pull request merged**